### PR TITLE
Add Common Cocoapods doc

### PR DESCRIPTION
**Pull request by alanscarpa**

### CODE REVIEW

File: ios/common_cocoapods.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- [Genome](https://github.com/LoganWright/Genome)
+- [SwiftyJSON](https://github.com/SwiftyJSON/SwiftyJSON)
+
+## Dates
+- [AFDateHelper](https://github.com/melvitax/DateHelper)
+- [NSDate-Escort](https://github.com/azu/NSDate-Escort)
+
+## Image Downloading & Caching
+- [Kingfisher](https://github.com/onevcat/Kingfisher)
+- [SDWebImage](https://github.com/rs/SDWebImage)
```
Comment (Benuuu) [inline ios/common_cocoapods.md:42]: Do people also use AlamofireImage?  it basically has the same API as SDWebImage

File: common_cocoapods.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Common CocoaPods
```
Comment (Benuuu) [inline common_cocoapods.md]: you appear to have duplicated this file.  Please this one.

**Review by Benuuu — APPROVED** [review]
Fix the duplicate file. then :shipit:

**Comment by paulrolfe** [comment]
I also like FXBlurView and should RxSwift be here too?

File: ios/common_cocoapods.md (hunk 1 above)
Comment (brightredchilli) [reply ios/common_cocoapods.md:42]: I think maye we should just add this

File: ios/common_cocoapods.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- [Alamofire](https://github.com/Alamofire/Alamofire)
+
+## Persistence
+- [Realm](https://github.com/realm/realm-cocoa)
+
+## Keyboard Handling
+- [TPKeyboardAvoiding](https://github.com/michaeltyson/TPKeyboardAvoiding)
+- [IQKeyboardManager](https://github.com/hackiftekhar/IQKeyboardManager)
+
+## Analytics
```
Comment (intrepidmatt) [inline ios/common_cocoapods.md:15]: Add localytics?

File: ios/common_cocoapods.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- [Fabric](https://get.fabric.io/)
+- [Crashlytics](http://try.crashlytics.com/)
```
Comment (intrepidmatt) [inline ios/common_cocoapods.md]: Fabric and crashlytics are essentially the same thing

**Comment by brightredchilli** [comment]
Also add PermissionsKit

File: ios/common_cocoapods.md (hunk 3 above)
Comment (alanscarpa) [reply ios/common_cocoapods.md:15]: will do.

File: ios/common_cocoapods.md (hunk 4 above)
Comment (alanscarpa) [reply ios/common_cocoapods.md]: Ok..I'll remove crashlytics.  Unless you prefer to keep that link here instead?

File: ios/common_cocoapods.md (hunk 1 above)
Comment (alanscarpa) [reply ios/common_cocoapods.md:42]: done.

**Review by brightredchilli — APPROVED** [review]

**Pull request merged**